vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -44,5 +44,10 @@ namespace Microsoft.ReactNative {
 
     // Call JavaScript module event. It is a specialized CallJSFunction call where method name is always 'emit'.
     void EmitJSEvent(String eventEmitterName, String eventName, JSValueArgWriter paramsArgWriter);
+
+    Boolean UseWebDebugger { get; };
```

Comment:
Out of curiosity, why just these vs exposing all options?

Reply:
I agree. Could we expose all ReactSettings instead?

---
In reply to: [497641072](https://github.com/microsoft/react-native-windows/pull/6135#discussion_r497641072) [](ancestors = 497641072)

Reply:
Done.  (Excluding some that are basically duplicates of others that we are looking to remove in future releases)